src/tests/JIT/HardwareIntrinsics/Arm/Shared/Helpers.cs

```diff
@@ -5218,6 +5218,82 @@ public static float MultiplyExtended(float op1, float op2)
             }
         }
 
+        public static float FPExponentialAccelerator(uint op1)
```

Comment:
this should also need handling for `double`?

Reply:
There is a `double` variant farther down the `Helpers.cs` file, on line 5341. I put them separately to match the current format of the file, though I can move them next to each other if it's easier to read.